File: cpp/344.reverseString.cpp

class Solution {
public:
    void reverseString(std::vector<char>& s) {
        int low = 0, high = s.size()-1;
        for (;low < high;low++,high--) {
            std::swap(s[low],s[high]);
        }
    }
};
#include <iostream>
int main() {

}

File: py/344.reverseString.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        n = len(s)
        l = 0
        r = n-1
        while l < r:
            temp = s[l]
            s[l] = s[r]
            s[r] = temp
            l += 1
            r -= 1